src/server/pfs/server/driver_v2.go

```diff
@@ -961,3 +970,45 @@ func (d *driverV2) deleteCommit(txnCtx *txnenv.TransactionContext, userCommit *p
 
 	return nil
 }
+
+func (d *driverV2) createTempFileSet(server pfs.API_CreateTempFileSetServer) (string, *time.Time, error) {
+	ctx := server.Context()
+	opts := []fileset.UWriterOption{fileset.WithWriterOption(fileset.WithTTL(30 * time.Minute))}
+	id := uuid.NewWithoutDashes()
+	var commit *pfs.Commit
+	// TODO: within commit?
```

Comment:
I think we wanted it in a commit so that the rpcs that access files would work out of the box. Is the concern around the cleanup of the commit? I guess that would not be captured in this model. I think we are eventually going to have a background process that cleans up commits for data retention purposes and maybe it could be what catches the commits associated with cleaned up temporary filesets?